website/public/js/filters/taskOrdering.js

```diff
@@ -22,6 +22,13 @@ angular.module('habitrpg')
       for (var i = 0; i < input.length; i++) {
         if (term.test(input[i].text) || term.test(input[i].notes)) {
           result.push(input[i]);
+        }else if (input[i].checklist) {
+          var checkListLen = input[i].checklist.length;
+          for (var j = 0; j < checkListLen; j++) {
+            if ( term.test(input[i].checklist[j].text) ) {
+              result.push(input[i]);
```

Comment:
When using a for loop, I think you'll need a break after this line. Otherwise you risk pushing the task multiple times.